internal/resources/templates/generic_results.go

```diff
@@ -24,11 +24,27 @@ const GenericResults = `
 			<div class="ui tabs divider"></div>
 		</div>
 		<div class="ui container">
+		<div class="ui grid">
+		<div class="column" style="width:65%">
```

Comment:
Switch the order of "content" and "history" `div` and change the width to 80% content and 20% history.